packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -154,12 +154,16 @@ export class SummaryManager implements IDisposable {
     }
 
     private readonly refreshSummarizer = () => {
+        function safeAlert(m: string) { try { alert(m); } catch { console.log("FAILED TO ALERT"); } }
+
         // Transition states depending on shouldSummarize, which is a calculated property
         // that is only true if this client is connected and is the elected summarizer.
         const shouldSummarizeState = this.getShouldSummarizeState();
         switch (this.state) {
             case SummaryManagerState.Off: {
                 if (shouldSummarizeState.shouldSummarize) {
+                    // eslint-disable-next-line max-len
+                    safeAlert("YOU'VE GOT THE SUMMARIZER!\n\nYou're the only client who will see most alerts when an Inactive Object is used");
```

Comment:
Not sure we need this to be here. We can just add it to GarbageColletor's constructor. If it's the summarizer client, display this alert.
This code will eventually end up spawning a new container runtime and garbage collector anyway.

Reply:
I'm leaving it for parity with the alert a few lines down.

Reply:
Okay. Although note that the second alert is redundant with the one on GarbageCollector::dispose because it will shut down the client. Basically, at start, a whole new client is created and on stop, the client is shut down.

Reply:
Removed now that we're going to broadcast messages to all clients via DebugBus